packages/cms-lib/fileManager.js

```diff
@@ -143,13 +144,16 @@ async function downloadFile(portalId, file, dest, options) {
  * @param {number} portalId
  * @param {string} folderPath
  */
-async function getAllPagedFiles(portalId, folderPath) {
+async function getAllPagedFiles(portalId, folderId, { includeArchived }) {
   let totalFiles = null;
   let files = [];
   let count = 0;
   let offset = 0;
   while (totalFiles === null || count < totalFiles) {
-    const response = await getFilesByPath(portalId, folderPath, { offset });
+    const response = await getFiles(portalId, folderId, {
+      offset,
+      archived: includeArchived ? 1 : 0,
```

Comment:
Might be worthwhile to use booleans until we get to the API helper since we already need code to handle the option.